src/lib/Map.zn

```diff
@@ -6,7 +6,7 @@ library Map {
     static integer ENV_BETA = 102;
     static integer ENV_TEST = 103;
 
-    private static integer currentEnv = ENV_<%= @opt[:env].upcase %>;
+    private static integer currentEnv = ENV_TEST;
 
     static method getEnv() -> integer {
       return currentEnv;
```

Comment:
Lets remove `currentEnv` and `getEnv`, as well as the `ENV_*` variables. The same thing can be accomplished with debug mode/debug blocks.